src/cljs/athens/keybindings.cljs

```diff
@@ -40,7 +40,7 @@
         value (.. target -value)
         event {:key key :key-code key-code :target target :value value}
         modifiers (modifier-keys e)
-        [start end] (get-end-points e)
+        [start end] (get-end-points target)
         selection (getText target)
         head (subs value 0 start)
         tail (subs value end)]
```

Comment:
Can probably just create a `destruct-key-down` and `destruct-target` or `destruct-mouse-down`, which refactors some of the `tail` `head` `start` `end` stuff.